apps/vmq_enhanced_auth/src/vmq_enhanced_auth.erl

```diff
@@ -223,42 +223,65 @@ load_from_list(List) ->
     parse_acl_line(F(F, read), all),
     del_aged_entries().
 
+get_topic_label(Rest) ->
+    case binary:split(Rest, <<"label ">>) of
+        [Topic, Label] ->
+            case string:lexemes(Label, " ") of
+                [_] ->
+                    {Topic, string:trim(Label)};
+                _ ->
+                    error_logger:error_msg("can't add multiple label value!"),
```

Comment:
Make it a warning and only consider the first element instead of none.

Reply:
Done